src/utiltime.cpp

```diff
@@ -46,6 +46,15 @@ int64_t GetTimeMicros()
     return now;
 }
 
+int64_t GetTimeBenchmark()
+{
+    int64_t now = (boost::posix_time::microsec_clock::universal_time() -
+                      boost::posix_time::ptime(boost::gregorian::date(1970, 1, 1)))
+                      .total_microseconds();
```

Comment:
for benchmarks you typically want to use steady_clock.  This won't stop my merge, because these are currently for human consumption (and we can throw out weird results) but take a look at steady_clock.